src/pages/DataSharingLanguageTool.js

```diff
@@ -0,0 +1,232 @@
+  const [otherText, setOtherText] = useState();
+  const [nmds, setNmds] = useState(false);
+  const [gso, setGso] = useState(false);
+  const [pub, setPub] = useState(false);
+  const [col, setCol] = useState(false);
+  const [irb, setIrb] = useState(false);
+  const [gs, setGs] = useState(false);
+  const [mor, setMor] = useState(false);
+  const [npu, setNpu] = useState(false);
+  const [sdsl, setSdsl] = useState("");
+
+  const isTypeOfResearchValid = () => {
+    return (general || hmb || (diseases && !isNil(ontologies)) || other );
+  };
+
+  const generate = () => {
+    isTypeOfResearchValid() ?
+      generateHelper()
+      : Notifications.showError({text: "Please complete question 1"});
+  };
+
+  const generateHelper = () => {
+    const darInfo = {
+      general: general, diseases: diseases, ontologies: ontologies, other: other, otherText: otherText,
+      hmb: hmb, methods: nmds, forProfit: !npu
+    };
+    const summaries = [];
+    const dataUse = (DataUseTranslation.translateDarInfo(darInfo));
+    dataUse.primary.forEach((summary) => {
+      summaries.push(summary.description);
+    });
+    dataUse.secondary.forEach((summary) => summaries.push(summary.description));
+    setSdsl(summaries.join(" "));
+  };
+
+  return (
+    div({style: {...Styles.PAGE, color: '#1f3b50' }}, [
+      div({style: {...Styles.TITLE, marginTop: '3.5rem'}}, [
+        "Standardized Data Sharing Language Tool"
+      ]),
+      div({style: {...Styles.SMALL, marginTop: '1rem'}}, [
+        "This tool is made publicly available by the DUOS team for anyone" +
+        " interested in leveraging standardized data sharing language in their" +
+        " consent forms. The tool leverages the Global Alliance for Genomics " +
+        "and Health’s (GA4GH) companion standards of the Data Use Ontology (DUO) " +
+        "and Machine Readable Consent Guidance (MRCG). The DUO is a structured " +
+        "vocabulary describing permitted data uses and the MRCG is a suggested " +
+        "representation of those uses in consent form language. This tool enables" +
+        " users to easily define what types of data use they would like permitted" +
+        " in their consent forms and then suggests corresponding text for the " +
+        "consent form below, based on the MRCG."
+      ]),
+      div({className: 'form-group', style: {marginTop: '1rem'}}, [
+        label({style: Styles.MEDIUM}, [
+          '1. Choose the permitted data uses for your study\'s data ', br(),
+          span({style: Styles.MEDIUM_DESCRIPTION}, ['First, you must determine what type of secondary use is permitted for you study\'s data.' +
+          ' You do this by selecting one of the options in the following section:']),
+        ]),
+        div({}, [
+          RadioButton({
+            value: 'general',
+            defaultChecked: general,
+            onClick: () => {
+              setGeneral(!general), setHmb(false), setDiseases(false), setOther(false);
+            },
```

Comment:
The `setDiseases` method doesn't clear out existing selections (it only modifies state) so the ontology text box doesn't clear if you enter something and then select something else.

Reply:
Same feedback for `setOther`